Python/Product/PythonTools/PythonTools/Repl/PythonInteractiveEvaluator.cs

```diff
@@ -134,7 +134,17 @@ partial class PythonInteractiveEvaluator :
                 if (config == null) {
                     _analyzer = _serviceProvider.GetPythonToolsService().DefaultAnalyzer;
                 } else {
-                    _analyzer = new VsProjectAnalyzer(_serviceProvider, interpreterService.FindInterpreter(config.Interpreter.Id));
+                    Build.Evaluation.Project projectFile = null;
+                    var moniker = ProjectMoniker ??
+                        interpreterService.GetProperty(config.Interpreter.Id, "ProjectMoniker") as string;
```

Comment:
I'd prefer to have a named constant for this literal, since it's used in more than one place.